src/components/layout/Booking/BookingForm/BookingFormContent/__specs__/BookingFormContent.spec.jsx

```diff
@@ -91,24 +91,28 @@ describe('bookingFormContent', () => {
 
     it('should display a notice regarding the withdrawal period', () => {
```

Comment:
```suggestion
    it('should display a notice regarding the cancellation period', () => {
```
Idem, j'aimais bien withdrawal mais je préfère encore qu'on ai une unité dans la traduction du terme